src/brs/util/StringUtils.java

```diff
@@ -0,0 +1,21 @@
+package brs.util;
+
+public class StringUtils {
```

Comment:
A very good candidate for a unit test ;-) (just to bother you)

Reply:
I'll need to do a ton of unit tests soon anyway... ;)